.github/workflows/run-tests.yml

```diff
@@ -1,6 +1,7 @@
 name: Tests
 
 on:
+  workflow_dispatch:
```

Comment:
Went ahead and added this trigger since it is nice to be able to run CI on command